src/lib.rs

```diff
@@ -141,6 +153,14 @@ pub mod signrpc {
     tonic::include_proto!("signrpc");
 }
 
+pub mod looprpc {
+    tonic::include_proto!("looprpc");
+}
+
```

Comment:
Why only `looprpc` and `frdrpc` up here, why no `swaprpc`? Or is that included in the looprpc?

Reply:
It's included in looprpc